docs/how-to-guides/use-init-flags.md

```diff
@@ -0,0 +1,186 @@
+---
+id: use-init-flags
+title: How to use init flags with Docker
+sidebar_label: Use init flags with Docker
+---
+
+With `init-file` and `init-data-file` configuration flags, you can execute queries from the CYPHERL file which need to be executed before or immediately after the Bolt server starts, respectively. The CYPHERL file, that the `init-file` flag is pointing to, is usually used to create users and set their passwords, so that no one except for the authorized users can't access data in the first run. The CYPHERL file, that the `init-data-file`is pointing to, is usually used to populate the database. If you plan to run Memgraph in Docker, then you have to make sure that the files `init-file` and `init-data-file` configuration flags are referring to are inside the container before Memgraph starts. In Docker, files can't be directly copied into a container before it's started because the filesystem of the container doesn't exist until it's actually running. However, you can tackle this by using a Dockerfile.
+
+In this guide you will learn how to:
+- [**Use init-file flag with Docker**](#use-init-file-flag-with-docker)
+- [**Use init-data-file flag with Docker**](#use-init-data-file-flag-with-docker)
+
+## Use init-file flag with Docker
+
+### 1. Create all necessary files
+
+First, create a directory called `my_init_test` and create `auth.cypherl` and Dockerfile inside it.
+
+Here is the content of `auth.cypherl` file:
+
+```
+CREATE USER memgraph1 IDENTIFIED BY '1234';
+```
+
+Here is how you should define a Dockerfile:
+
+```bash
+FROM memgraph/memgraph:latest
+
+USER root
+
+COPY auth.cypherl /usr/lib/memgraph/auth.cypherl
+
+USER memgraph
+```
+
+The above Dockerfile will build an image based on `memgraph/memgraph:latest` image. For other images, [check Memgraph's Docker Hub](https://hub.docker.com/u/memgraph). Next, switch to user `root` to be able to copy your local file to the container where Memgraph will be run. Due to the permissions set, it is recommended to copy it to `/usr/lib/memgraph/` or any subfolder within that folder. In the end, don't forget to switch user back to `memgraph`.
+
+### 2. Build the Docker image
+
+Open the terminal, place yourself in the `my_init_test` directory and build the image called `my_image` with the following command:
+
+```
+docker build -t my_image .
+```
+
+### 3. Run the Docker image
+
+Once you've built the Docker image, you can run it with the flag `init-file` set to the appropriate value:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 my_image --init-file=/usr/lib/memgraph/auth.cypherl
+```
+
+To learn how to set the flags with other Docker images, refer to [configuration flags guide](https://memgraph.com/docs/memgraph/how-to-guides/config-logs).
```

Comment:
```suggestion
To check how to set the flags with other Docker images, refer to [the configuration guide](https://memgraph.com/docs/memgraph/how-to-guides/config-logs).
```

how will this resource help

Reply:
It would be better to link to config reference guide again. I will fix this and let me know if it's not good.